setup.cfg

```diff
@@ -21,8 +21,8 @@ classifiers =
   Programming Language :: Python :: 3.7
   Operating System :: OS Independent
```

Comment:
Are you going to add Mac and more unix/linux system tests to back this up?

Reply:
Updated as linux / mac / windows.  The mac tests fail on github actions due to a configuration issue (#149).  Willing to keep the mac descriptor as I run locally on a mac.